modules/ROOT/pages/enabling-api-catalog.adoc

```diff
@@ -0,0 +1,33 @@
+= Enabling API Catalog
+
+Enable a connected Salesforce organization to access and sync Anypoint Platform APIs into the Salesforce API Catalog and expose them as invocable actions. This capability is enabled in Anypoint Platform Access Management by an Anypoint Platform organization administrator.  
+
+[[enable-disable-api-catalog]]
+== Enable or Disable API Catalog in Anypoint Platform
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. Click *Salesforce*.
+. In *Capabilities Managed in Anypoint Platform*, click *Settings* for *Make MuleSoft Anypoint Platform APIs Available in Salesforce*.
+. In *Settings for Anypoint APIs in Salesforce*, click *Continue*.
+. In *Settings*, select the Anypoint Platform business groups that allow the Salesforce organization to import Anypoint Platform APIs into its API Catalog, and click *Save Settings*.
++
+[NOTE]
+Within the selected business groups, you can import only APIs that have specs published in Exchange with active instances into Salesforce.
+. In *Capabilities Managed in Anypoint Platform*, switch between *Enabled* or *Disabled* in *Make MuleSoft Anypoint Platform APIs Available in Salesforce*. 
++
+image:connected-salesforce-org-api-catalog-enabled.png["Connected Salesforce organization with both Invocable Actions and API Catalog enabled"]
++
+The Salesforce organization administrator can then sync APIs into their catalog. 
++
+For details about syncing APIs and using API Catalog in Salesforce, see xref:general::api-catalog-in-salesforce.adoc[].
+. After the APIs are synced, the Salesforce administrator selects individual API instances and operations and converts them into invocable actions so Salesforce developers can access them.
++
+A client ID is created.
```

Comment:
We might want to have Yining Zhao and/or Mariana Zurita review this. Salesforce admins create connections for the API instances either as part of activating actions or for existing actions, and these connections handle the authentication between Anypoint Platform and Salesforce based on credentials and permission sets. The Salesforce admin adds users (here called Salesforce developers) to permission sets. API Catalog can detect through the connection functionality when access needs to be requested, when access is rejected or revoked, and when a client contract has been deleted. API Catalog also has options for selecting SLA tiers. I'm still getting familiar with the details on this.